data/config/TCPairsConfig_default

```diff
@@ -137,6 +137,34 @@ watch_warn = {
    time_offset = -14400;
 }
 
+//
+// Diagnostics to be extracted
+//
+diag_name = [];
+
+//
+// Unit conversions to be applied based on diagnostic names and units
+//
+diag_convert_map = [
+   { source = "TCDIAG";
+     key = [ "(10C)", "(10KT)", "(10M/S)" ];
+     convert(x) = x / 10; },
+
+   { source = "LSDIAG_RT";
+     key = [ "CSST", "RSST", "DSST", "DSTA", "XDST", "XNST", "NSST", "NSTA",
```

Comment:
Should LAT and LON be in this list? Are are these used for matching with the track object and not included? They are also divided by 10.

Reply:
The LAT and LON lines are parsed as part of the "header" information. In fact, if they are missing from the input diag file, it should error out. As such, dividing LAT and LON by 10 is done automatically without any user option to do otherwise. I think that behavior makes sense.

Reply:
@jvigh, OK, as requested I just pushed changes to let LAT and LON be included in the values written to the TCDIAG line. You can inspect those changes in [this commit](https://github.com/dtcenter/MET/commit/56f97c8aa2d459cec0ed5856fa85c75322e7b2c6).

Once you're satisfied with these changes, please actually add your "approval" on this PR. That adds a little green checkmark by your name is how you tell me that you're ready to proceed with these changes.